doc/spec/Types.md

```diff
@@ -123,7 +123,13 @@ The only possible values for the Void type are `null` and `undefined`. The Void
 
 *NOTE: We might consider disallowing declaring variables of type Void as they serve no useful purpose. However, because Void is permitted as a type argument to a generic type or function it is not feasible to disallow Void properties or parameters*.
 
-### The Null Type { #the-null-type }
+### Unit Types { #unit-types }
+
+Unit types are types with a single value. The unit types in TypeScript are `null`, `undefined`, string literal types, number literal types and boolean literal types.

```

Comment:
Oxford comma, enum member types?

Reply:
> boolean literal types
Perhaps we could reduce the layers of abstraction in this statement and just say "the `true` and `false` types"

Reply:
@weswigham consistency "string literal types, number literal types" is easier to understand, at least in normal usage, and I think the same holds true in the spec.

@DanielRosenwasser fixed